allure-python-commons/src/_allure.py

```diff
@@ -70,6 +70,10 @@ def id(id):
     return label(LabelType.ID, id)
 
 
+def manual():
+    return label(LabelType.MANUAL, True)
```

Comment:
use variable for static call:
`manual = label(LabelType.MANUAL, True)`

Reply:
Oh, I see.

I was ready to push commit with Dynamic usage. Something like


```
def manual(fn):
    def inner():
        return Dynamic.label(LabelType.MANUAL, True)
    return inner

```

Variable solution works fine. Didn't think about it. Thanks!

Reply:
@skhomuti Well, actually variable solution does not work for some reason. It does not mark test with ALLURE_MANUAL label. May be dynamic label is better way?

Reply:
:( i'll think and return to you later

Reply:
@betapl3b take a look on the last commits